src/lib/swapper/LifiSwapper/LifiSwapper.ts

```diff
@@ -67,14 +73,43 @@ export class LifiSwapper implements Swapper<EvmChainId> {
    * Builds a trade with definitive rate & txData that can be executed with executeTrade
    **/
   async buildTrade(input: BuildTradeInput): Promise<LifiTrade> {
-    return await buildTrade(input, this.lifiAssetMap, this.lifiChainMap, this.lifiBridges)
+    return await buildTrade(input, this.lifiAssetMap, this.lifiChainMap)
   }
 
   /**
    * Get a trade quote
    */
-  async getTradeQuote(input: GetEvmTradeQuoteInput): Promise<TradeQuote<EvmChainId>> {
-    return await getTradeQuote(input, this.lifiAssetMap, this.lifiChainMap, this.lifiBridges)
+  async getTradeQuote(input: GetEvmTradeQuoteInput): Promise<LifiTradeQuote> {
+    const minimumCryptoHuman = getMinimumCryptoHuman(input.sellAsset)
+    const minimumSellAmountBaseUnit = toBaseUnit(minimumCryptoHuman, input.sellAsset.precision)
```

Comment:
Loss of precision, we don't want to use a human amount to go to a base unit. Though it is *actually* fine here because `MIN_AMOUNT_THRESHOLD_USD_HUMAN` is an integer, but we still want this to be `minimumCryptoPrecision` - values shouldn't be stored as human